adapters/wpcom-vip.php

```diff
@@ -6,9 +6,25 @@
 
 class ES_WP_Query extends ES_WP_Query_Wrapper {
 	protected function query_es( $es_args ) {
-		if ( function_exists( 'es_api_search_index' ) ) {
-			return es_api_search_index( $es_args, 'es-wp-query' );
+		if ( ! function_exists( 'es_api_search_index' ) ) {
+			return new WP_Error( 'vip-es-api-missing', 'Missing es_api_search_index method' );
```

Comment:
@mboynes Previously it was failing silently. I noticed that `set_posts()` checks for `is_wp_error()` so this shouldn't introduce any regression.